TradingBotTests/MarketDataSourceTests.cs

```diff
@@ -48,7 +48,7 @@ public async Task ShouldCorrectlyReturnPrices()
     {
         SetUpResponses();
 
-        var result = await _marketDataSource.GetPricesAsync(DateOnly.MinValue, DateOnly.MaxValue);
+        var result = await _marketDataSource.GetPricesAsync(DateOnly.MinValue, new DateOnly(2023, 12, 19));
```

Comment:
We could change end date to earlier value, so that we don't need to download the csv with fear/greed index from GitHub.